apps/etterlatte-beregning/src/main/kotlin/beregning/BeregningRepository.kt

```diff
@@ -304,15 +309,21 @@ private object Queries {
         SELECT *
         FROM overstyr_beregning 
         WHERE sak_id = :sakId
+        and status = '${OverstyrBeregningStatus.GYLDIG.name}'
         """.trimIndent()
 
     val opprettOverstyrBeregning =
         """
-        INSERT INTO overstyr_beregning (sak_id, beskrivelse, tidspunkt)
-        VALUES (:sakId, :beskrivelse, :tidspunkt)
+        INSERT INTO overstyr_beregning (sak_id, beskrivelse, tidspunkt, status)
+        VALUES (:sakId, :beskrivelse, :tidspunkt, :status)
         """.trimIndent()
 }
 
+enum class OverstyrBeregningStatus {
+    GYLDIG,
```

Comment:
Er `GYLDIG` eller `UGYLDIG` riktig begreper å bruke her tro? Ser for meg at en sak enten har aktivert overstyrt beregning eller ikke. Kunne `AKTIV`, `IKKE_AKTIV` vært tydeligere?

Reply:
Ja, det er kanskje tydelegare på kva konsekvensen av feltet er. Godt forslag.